### PR TITLE
Create model and relationship.

### PR DESCRIPTION
**Pull request by ducls-2223**

### CODE REVIEW

**Comment by ducls-2223** [comment]
ready

File: app/Models/Activity.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Activity extends Model
+{
+    protected $fillable = [
+        'name',
+        'user_id',
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo('App\Models\User');
```
Comment (tuannt-0910) [inline app/Models/Activity.php]: sử dụng User::class

File: app/Models/Category.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Category extends Model
+{
+    protected $fillable = [
+        'name',
+        'parent_id',
+    ];
+
+    public function categories()
+    {
+        return $this->hasMany('App\Models\Category', 'parent_id');
```
Comment (tuannt-0910) [inline app/Models/Category.php]: tương tự

**Comment by tuannt-0910** [comment]
tách 1 pull model, 1 pull migrate

**Comment by ducls-2223** [comment]
ready

**Comment by ducls-2223** [comment]
ready

**Pull request merged**